File: official/docs/java/current/carrier-accounts/delete.java
package carrier_accounts;

import com.easypost.exception.EasyPostException;
import com.easypost.service.EasyPostClient;

public class Delete {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient(System.getenv("EASYPOST_API_KEY"));

        client.carrierAccount.delete("ca_...");
    }
}

File: official/docs/csharp/current/carrier-accounts/delete.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;
using EasyPost.Models.API;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            CarrierAccount carrierAccount = await client.CarrierAccount.Retrieve("ca_...");

            await client.CarrierAccount.Delete(carrierAccount.Id);
        }
    }
